python/cuda_parallel/tests/test_reduce_api.py

```diff
@@ -11,19 +11,21 @@
 # example-end imports
 
 
-def test_device_reduce():
-    # example-begin reduce-min
-    def op(a, b):
-        return a if a < b else b
+# example-begin reduce-min
+def min_op(a, b):
+    return a if a < b else b
+# example-end reduce-min
```

Comment:
important: the `test_*_api` files are used for documentation purposes. They illustrate usage of API. Sphinx essentially copy-pastes code between the `example-begin reduce-min` and `example-begin reduce-min` labels. [Here's](https://nvidia.github.io/cccl/cuda_parallel/) an example how it looked like. This cange makes it so that our reference API documentation only contains `min_op` definition. Let's revert this change.

Reply:
Done. Thanks for the explanation!

I only kept some minor changes.